community/cypher/runtime-util/src/test/scala/org/neo4j/cypher/FakeClock.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher
+
+import java.time.Instant
+
+import org.neo4j.time.Clocks
+
+trait FakeClock {
```

Comment:
Would it make sense to add this here?
```
self: GraphDatabaseTestSupport  =>
```

Reply:
Maybe. It wasn't needed, so I didn't bother with the extra typing.